From 2a230250e61dd40735a338f5264a3c357f5c1156 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20=C4=8Cerm=C3=A1k?= <dcermak@suse.com>
Date: Tue, 1 Nov 2022 15:53:19 +0100
Subject: [PATCH 12/12] Relax rspec dependency to ~> 3.11

No need to be so overly strict, this is a test dependency, so failures will
surface immediately.
---
 vagrant.gemspec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vagrant.gemspec b/vagrant.gemspec
index 8fde90bad..4f2cb20ab 100644
--- a/vagrant.gemspec
+++ b/vagrant.gemspec
@@ -45,7 +45,7 @@ Gem::Specification.new do |s|
   # Constraint rake to properly handle deprecated method usage
   # from within rspec
   s.add_development_dependency "rake", "~> 13.0"
-  s.add_development_dependency "rspec", "~> 3.11.0"
+  s.add_development_dependency "rspec", "~> 3.11"
   s.add_development_dependency "rspec-its", "~> 1.3.0"
   s.add_development_dependency "fake_ftp", "~> 0.3.0"
   s.add_development_dependency "webrick", "~> 1.7.0"
-- 
2.38.1

